frontend/src/components/JumpBar.js

```diff
@@ -44,17 +44,19 @@ const JumpBar = (props) => {
   const history = useNavigate();
   const selectRef = useRef();
 
-  const search = useMemo(() => debounce(150, query => {
-    setValue(null)
-    setIsFetching(true)
-    withToken(token, api.query.search)(query)
-      .then(response => { setItems(response.data) })
-      .catch(err => {
-        if (err.name === 'AbortError')
-          return
-        setError(err.message)
-      })
-      .then(() => setIsFetching(false))
+  const search = useMemo(() => debounce(150, async query => {
```

Comment:
No matter how fast I type, a call to the backend is being triggered on every character I type. 150ms for the debounce time is too small. Let's change it to 500ms.